packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1239,6 +1240,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
                     cookieJar,
                 });
             },
+            getKeyValueStore: KeyValueStore.open,
```

Comment:
IMO we should pass the crawler's `config` instance here too (just like you do above in the `Dataset.open` call above). The storage implementation mismatch for different storage types is a _really_ annoying thing to deal with 😅

Reply:
Good point, thanks!